src/components/topbar/index.tsx

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Box, AppBar, Toolbar, IconButton, Badge, Tooltip } from '@mui/material';
 import { LeftSide, RightSide, StyledContainer } from './styled';
 import {
   Person as PersonIcon,
   Notifications as NotificationsIcon,
   GridOnOutlined as GridOnOutlinedIcon,
 } from '@mui/icons-material';
+import { ProfileModal } from 'components/modal/profile';
+import { InteractiveComponentsContext } from 'contexts/interactiveComponentsContext';
 
 export default function TopBar() {
+  const { openProfileModal } = useContext(InteractiveComponentsContext);
```

Comment:
this function doesn't even exist in the InteractiveComponentsContext

Reply:
This change wasn't pushed at that time. This should be addressed by now.